docs/source/ruletypes.rst

```diff
@@ -58,6 +58,20 @@ Rule Configuration Cheat Sheet
 +--------------------------------------------------------------+           |
 | ``kibana4_end_timedelta`` (time, default: 10 min)            |           |
 +--------------------------------------------------------------+           |
+| ``generate_kibana_discover_url`` (boolean, default False)    |           |
++--------------------------------------------------------------+           |
+| ``kibana_discover_app_url`` (string, no default)             |           |
++--------------------------------------------------------------+           |
+| ``kibana_discover_version`` (string, no default)             |           |
++--------------------------------------------------------------+           |
+| ``kibana_discover_index_pattern_id`` (string, no default)    |           |
++--------------------------------------------------------------+           |
+| ``kibana_discover_columns`` (list of strs, default _source)  |           |
++--------------------------------------------------------------+           |
+| ``kibana_discover_from_timedelta`` (time, default: 10 min)   |           |
++--------------------------------------------------------------+           |
+| ``kibana_discover_to_timedelta`` (time, default: 10 min)     |           |
++--------------------------------------------------------------+           |
```

Comment:
I envision that most users will configure as such:

Global config file: (or next to es settings)

* `kibana_discover_app_url`
* `kibana_discover_version`
* `kibana_discover_index_pattern_id` if only using a single index

Shared import file:

* `kibana_discover_index_pattern_id` if multiple indexes

Local rule file:

* `generate_kibana_discover_url`
* `kibana_discover_columns`